paypal/standard/helpers.py

```diff
@@ -1,16 +1,25 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 import hashlib
-
+import sys
 from django.conf import settings
 from django.utils.encoding import smart_str
-
 from paypal.utils import warn_untested
 
+if sys.version_info[0] >= 3:
+    unicode = str
+
+
+def _get_hash(salt, raw_password):
+    hashed = smart_str(salt) + smart_str(raw_password)
+    if sys.version_info[0] >= 3:
+        hashed = hashed.encode()
+    return hashed
+
```

Comment:
This is kind of confusingly named (it doesn't return a hash, but something that is going to be hashed later), and I think the best solution is to just inline it. 

In fact, looking at the way that `hashlib.sha1` behaves on Python 2, it accepts both bytes and unicode strings, while Python 3 only accepts bytes. So in fact all we need is an unconditional `encode()` call. However, it should specify an encoding like 'utf-8' as otherwise we could get exceptions on Python 2.